lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriApiService.java

```diff
@@ -250,6 +250,8 @@ public HTTPRequest buildFetchVerifiableCredentialRequest(
         var apiKey = getApiKey(criConfig, criOAuthSessionItem);
 
         var request = new HTTPRequest(HTTPRequest.Method.POST, criConfig.getCredentialUrl());
+        request.setHeader(
+                "Content-Type", null); // remove the default, no request body so we don't need
```

Comment:
`setHeader` uses varargs for the value, this will throw a compiler warning. 

```java
warning: non-varargs call of varargs method with inexact argument type for last parameter;
        request.setHeader("Content-Type", null);
                                          ^
  cast to String for a varargs call
  cast to String[] for a non-varargs call and to suppress this warning
1 warning
```

You will need to pass an empty string.